model/lifecycle.go

```diff
@@ -128,7 +129,17 @@ func AbortVersion(versionId string) error {
 		},
 		bson.M{"$set": bson.M{task.AbortedKey: true}},
 	)
-	return err
+	if err != nil {
+		return errors.Wrap(err, "error setting aborted statuses")
+	}
+	ids, err := task.FindAllTaskIDsFromVersion(versionId)
+	if err != nil {
+		return errors.Wrap(err, "error finding tasks by version id")
+	}
+	if len(ids) > 0 {
+		event.LogManyTaskAbortRequests(ids, evergreen.GithubPatchUser)
```

Comment:
Why is this GithubPatchUser? "mci" is probably more appropriate if we don't have a good option

Reply:
done